docs/user/explanations/framework-summary.rst

```diff
@@ -0,0 +1,63 @@
+Framework Summary
+=================
+
+Tickit is an event-based simulation framework allowing for the simulation of
+complex mutli-device systems.
+
+A tickit simulation consists of a scheduler and :doc:`components<components>`_, all of which
+communicate via a message bus. The scheduler keeps simulation time running and
+updates the components in the simulation when required.
+
+
+.. figure:: ../../images/tickit-overview-full.svg
+    :align: center
+
+
+Components
+^^^^^^^^^^
+Components are typically comprised of :doc:`devices <devices>`_ and :doc:`adapters <adapters>`_. Devices are created
+as required to provide the necessary simulated the behaviour, and adapters are
+interfaces between devices and any external input. Adapters allow us to
+influence the device from outside the simulation, such as using a TCP client to
+alter a parameter on a device.
```

Comment:
I think we need to somehow get across that a component is a "thing". In other words, an `EigerDevice` isn't an Eiger, neither is an `EigerAdapter`, an `EigerComponent` is the thing that actually corresponds to the Eiger in real life.

Reply:
I've reworded this to try and emphasise this.